File: dongyang_algo/day09/Bitwise1.c
#include <stdio.h>
#include <string.h>

int isEven(int n)
{
    return (n & 1);
}
int main()
{
    int n = 16;

    if (isEven(n) == 1)
        printf("Odd");
    else
        printf("Even");
    return 0;
}

File: dongyang_algo/day15/ch9/Bitwise1.java
package day15.ch9;

public class Bitwise1 {
    public static String solution(int n) {
        if ((n&1)==1) {
            return "Odd";
        } else {
            return "Even";
        }
    }

    public static void main(String[] args) {
        System.out.println(solution(11));
    }
}
